keras/src/utils/jax_layer.py

```diff
@@ -214,15 +235,16 @@ def __init__(
         self,
         call_fn,
         init_fn=None,
+        compute_output_shape_fn=None,
         params=None,
         state=None,
         seed=None,
         **kwargs,
     ):
-        if backend.backend() != "jax":
+        if backend.backend() not in ["jax", "tensorflow"]:
             raise ValueError(
-                "JaxLayer is only supported with the JAX backend. Current "
-                f"backend: {backend.backend()}"
+                "JaxLayer is only supported with the JAX or Tensorflow backend"
+                f". Current backend: {backend.backend()}"
```

Comment:
I saw you removed the equivalent error message in `FlaxLayer`. Can you do this so that it works properly for both:
```python
f"{self.__class__.__name__} is only supported with the JAX ..."
```